server/bundles/io.cloudbeaver.server/src/io/cloudbeaver/server/graphql/GraphQLEndpoint.java

```
@@ -241,24 +244,20 @@ private void executeQuery(
         if (operationName != null) {
             contextBuilder.operationName(operationName);
         }
-        {
-            String apiCall = operationName;
-            //            if (!CommonUtils.isEmpty(apiCall)) {
-            //                if (variables != null) {
-            //                    apiCall += " (" + variables + ")";
-            //                }
-            //            }
-            String sessionId = GraphQLLoggerUtil.getSessionId(request);
-            String userId = GraphQLLoggerUtil.getUserId(request);
-            String loggerMessage = GraphQLLoggerUtil.buildLoggerMessage(sessionId, userId, variables);
-            if (apiCall != null) {
-                log.debug("API > " + apiCall + loggerMessage);
-            } else if (DEBUG) {
-                log.debug("API > " + query + loggerMessage);
+        String apiCall = operationName;
+        LocalDateTime startTime = LocalDateTime.now();
+        ExecutionInput executionInput = contextBuilder.build();
+        ExecutionResult executionResult = null;
+        try {
+            executionResult = graphQL.execute(executionInput);
+        } catch (Exception e) {
+            throw e;
```

Why ignore this error? If error happens executionResult is always null and we don't have info about the errors.